google/ggrc-core
# GGRC-3800 Only digitals should be allowed to enter into Hotlist ID and Component ID fields

# Issue description

_Hotlist ID_ and _Component ID_ should contain only numbers.
Only digitals should be allowed for _Component ID_ and _Hotlist ID_ inputs

# Steps to test the changes

1. User goes to:
'Add / Edit Audit' pop-up;
'Add / Edit Assessment' pop-up;
'Add / Edit Assessment Template' pop-up.
2. User types letters or special characters.
_Actual result_: It is possible to type letters or special characters.
_Expected result_: It should be impossible to type letters or special characters. Only digitals should be allowed

# Solution description

Create new "**Numberbox**" component and apply it to Component ID and Hotlist ID inputs

# Sanity checklist




# PR Review checklist
